kubernetes/types.go

```diff
@@ -97,6 +97,11 @@ const (
 	AuthorizationPoliciesType     = "AuthorizationPolicy"
 	AuthorizationPoliciesTypeList = "AuthorizationPolicyList"
 
+	// Peer Authentications
```

Comment:
On types.go, it's missing to add PeerAuthentications to PluralType map.
(Lines 356-394)